homeassistant/components/homewizard_climate/__init__.py

```diff
@@ -0,0 +1,61 @@
+"""The Homewizard Climate integration."""
+from __future__ import annotations
+
+import logging
+
+from homewizard_climate_websocket.api.api import HomeWizardClimateApi
+from homewizard_climate_websocket.model.climate_device import HomeWizardClimateDevice
+from homewizard_climate_websocket.ws.hw_websocket import HomeWizardClimateWebSocket
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN, PASSWORD, USERNAME
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS: list[Platform] = [Platform.CLIMATE]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Homewizard Climate from a config entry."""
+
+    hass.data.setdefault(DOMAIN, {})
```

Comment:
Move this down a bit, after we are sure the init is successful. (above `hass.data[DOMAIN][entry.entry_id] = {}` for example

Reply:
Done.